packages/api-bindings/src/apollo/createPublicationTypePolicy.ts

```diff
@@ -44,9 +48,14 @@ export function createPublicationTypePolicy(): TypePolicy<R> {
         return existing ?? false;
       },
 
-      ownedByMe() {
-        // TODO: implement ownership check using active profile
-        return false;
+      ownedByMe: {
+        read(_: boolean | undefined, { readField }) {
+          const activeProfile = activeProfileVar();
+          if (activeProfile) {
+            return readField('id', readField('profile')) === activeProfile.id;
+          }
+          return false;
+        },
```

Comment:
Turns out we don't need this. I would omit it.

Reply:
Removed